GeoEntity.md

```diff
@@ -0,0 +1,280 @@
+- Request Structure:
+	- Target: http://localhost:8080/admin_area_1s (After adding to Routing layer, we update this)
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    
+	- Example URL: http://localhost:8080/admin_area_1s?country=us
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "long_name": "Alaska",
+        "short_name": "AK",
+        "level": 1,
+        "place_id": "ChIJG8CuwJzfAFQRNduKqSde27w",
+        "location": {
+          "lat": 64.2008413,
+          "lng": -149.4936733
+        },
+        "location_type": "APPROXIMATE",
+        "viewport": {
+          "northeast": {
+            "lat": 71.347066,
+            "lng": -129.9945563
+          },
+          "southwest": {
+            "lat": 51.220011,
+            "lng": 172.4573429
+          }
+        },
+        "formatted_address": "Alaska, USA"
+      }
+    ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Administrative Area Twos Request
+- Request Structure:
+	- Target: http://localhost:8080/admin_area_2s (After adding to Routing layer, we update this)
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted|
+    
+	- Example URL: http://localhost:8080/admin_area_2s?country=us&admin_area_one=ga
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "long_name": "Camden County",
+        "short_name": "Camden County",
+        "level": 2,
+        "place_id": "ChIJl6NPkwkb5YgRwGvHFXP6tYY",
+        "location": {
+          "lat": 30.898276,
+          "lng": -81.6035062
+        },
+        "location_type": "APPROXIMATE",
+        "viewport": {
+          "northeast": {
+            "lat": 31.169595,
+            "lng": -81.40354649999999
+          },
+          "southwest": {
+            "lat": 30.7114626,
+            "lng": -81.936988
+          }
+        },
+        "formatted_address": "Camden County, GA, USA"
+      }
+  ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Localities Request
+- Request Structure:
+	- Target: http://localhost:8080/localities (After adding to Routing layer, we update this)
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted|
```

Comment:
if I omit AA1, will I get e.g. all localities in the US? we should state this in the docs (for this and other endpoints). "If present, limits the result set to localities within the specified first-level administrative area. Otherwise, all localities within the specified country will be returned."